intel_extension_for_transformers/llm/runtime/graph/models/chatglm/chatglm2.cpp

```diff
@@ -200,7 +200,7 @@ static bool chatglm_model_eval_internal(model_context& lctx, const model_token*
         ne_set_name(attn_scores, "attn_scores");
         attn_scores = ne_scale_inplace(ctx0, attn_scores, ne_new_f32(ctx0, attn_scale));
 
-        if (n_total == 0 || !shift_roped_k) {
+        if (N > 1 || !shift_roped_k) {
```

Comment:
```N>1``` always true?

Reply:
False when N == 1

Reply:
N = 1 when input next token

Reply:
```n_total==0``` means first token, as well as ```N>1```, but ```N>1``` not always first token since input spliting?

Reply:
Yes, `N>1` for multi-round chatting, N will be bigger than 1 when new prompt from user